### PR TITLE
Adds global imageRegistry option

### PR DESCRIPTION
**Pull request by davividal**

### CODE REVIEW

File: charts/sonarqube-dce/values.yaml [hunk 1]
```diff
@@ -2,6 +2,17 @@
...
+## @param global.imageRegistry Global Docker image registry
+## @param global.imagePullSecrets Global Docker registry secret names as an array
+##
+global:
+  imageRegistry: ""
+  ## E.g.
+  ## imagePullSecrets:
+  ##   - myRegistryKeySecretName
+  ##
+  imagePullSecrets: []
```
Comment (jCOTINEAU) [inline charts/sonarqube-dce/values.yaml:14]: i dont see mention of this on the templates the imagePullSecret can probably be removed from the default values ?

**Review by jCOTINEAU — APPROVED** [review]
Should we add a bit more documentation on how this field works and how to migrate to it ? 
For example people already using something like
```
searchNodes:
  image:
    repository: repo.jfrog.io/sonarqube
    tag: 10.1.0.datacenter-search
```

Will have to rewrite a bit the current value with something like
```
global:
  imageRegistry: repo.jfrog.io
searchNodes:
  image:
    repository: sonarqube
    tag: 10.1.0-datacenter-search

```
As using the global registry field will have an impact on all images.

Also lacking Ticket number or NO-JIRA.

overall LGTM super cool improvment.

**Comment by pierre-guillot-gh** [comment]
Hey, don't forget to Squash the commits and mention the Jira ticket 👋

**Comment by jCOTINEAU** [comment]
Closing PR for now, as the sprint is over and non-trivial effort is still required to complete.

**Pull request closed without merging**